packaging/os/yumrepo.py

```diff
@@ -134,6 +174,14 @@
         are allowed. Substitution variables (e.g. C($releasever)) are honored
         here.
       - The list can also be a regular YAML array.
+  ip_resolve:
```

Comment:
I think I will leave it as it is. This module is no a man page for YUM. If the user wants to know more what the option mean, he should check the man page.